wandb/sdk/artifacts/artifact.py

```diff
@@ -1470,7 +1477,6 @@ def _add_local_file(
             staging_path = f.name
             shutil.copyfile(path, staging_path)
             os.chmod(staging_path, 0o400)
-
```

Comment:
Nit: this looks like a random unrelated change, revert.

Reply:
fixed, tried to fix other unintentional whitespace changes as well